aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -171,7 +171,7 @@ Enable the **IIS Management Console** and **World Wide Web Services**.
    
    To prevent the installer from installing x86 packages on an x64 OS, run the installer from an administrator command prompt with the switch `OPT_NO_X86=1`.
 
-1. Restart the system or execute **net stop was /y** followed by **net start w3svc** from a command prompt. Restarting IIS picks up a change to the system PATH made by the installer.
+1. Restart the system or execute **net stop was /y** followed by **net start w3svc** from a command prompt. Restarting IIS picks up a change to the system PATH, which is an environment variable, made by the installer.
```

Comment:
@shirhatti is this better than IISReset?

Reply:
According to this, they stopped supporting it ... https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/dd364067(v=ws.10)

Reply:
Yup. IISReset is no longer supported.